src/Purpose.html

```diff
@@ -5725,44 +5720,72 @@
+&lt;action&gt;I quickly look around for it- &lt;/action&gt;
 
 &lt;% if (window.story.getChoice(&quot;Chapter1&quot;, &quot;TiffBackstory&quot;) == &quot;Killing&quot;) { %&gt;
 
-\* Splat \*
+&lt;action&gt;SPLAT!&lt;/action&gt;
 
-I splutter, the walker just vomited blood or something all over my face! The weight of the walker lifts, I push it off and wipe my face so I can see again. I look at the walker... a knife is stuck square through the middle of its head, with the end poking out of its face just next to one of its eye sockets. I look to my right, %Tiffany% is stood there, hands to her chest, a blood splatter across her face quickly being washed away by the rain.
+&lt;action&gt;I splutter, the walker just vomited blood or something all over my face!&lt;/action&gt;
+&lt;action&gt;The weight of the walker lifts, I push it off and wipe my face so I can see again.&lt;/action&gt;
+&lt;action&gt;I look at the walker... a knife is stuck square through the middle of its head, with the end poking out of its face just next to one of its eye sockets.&lt;/action&gt;
+&lt;action&gt;I look to my right, %Tiffany% is stood there, hands to her chest, a blood splatter across her face quickly being washed away by the rain.&lt;/action&gt;
 
-&quot;%Tiffany%...&quot;, I start.
+&lt;speech.sarah&gt;&quot;%Tiffany%...&quot;&lt;/speech&gt;
+//will there be a character image of Tiffany here?
+&lt;character.two.confused.sad.neutral-backpack&gt;Tiffany&lt;/character&gt;
 
-\* Thunk \*
+&lt;action&gt; Thunk! &lt;/action&gt;
 
-I look over just in time to see a walker fall off the patio roof down onto the ground, with another close behind it.
+&lt;action&gt;I look over just in time to see a walker fall off the patio roof down onto the ground, with another close behind it.&lt;/action&gt;
 
-&quot;We need to go&quot;, I say back to %Tiffany%.
+&lt;speech.sarah&gt;We need to go.&lt;/speech&gt;
 
-[[[Get up]|C1P64]]
+&lt;choices&gt;[[[Get up]|C1P64]]&lt;/choices&gt;
 
 &lt;% } else { %&gt;
-There, it&#39;s just behind me, but out of reach. %Tiffany%, I need %Tiffany%&#39;s help! Where is she?
+&lt;action&gt;There, it&#39;s just behind me, but out of reach.&lt;/action&gt;
+&lt;action&gt;%Tiffany%, I need %Tiffany%&#39;s help! &lt;/action&gt;
+&lt;action&gt;Where is she?&lt;/action&gt;
 
-I look for her... there! She&#39;s hiding behind a tree near the edge of the garden.
+&lt;action&gt;I look for her... there! &lt;/action&gt;
+&lt;action&gt;She&#39;s hiding behind a tree near the edge of the garden.&lt;/action&gt;
 
-&quot;%Tiffany%- help!&quot;, I manage to shout.
+&lt;speech.sarah&gt;%Tiffany%- help!&lt;/speech&gt;
 
-Reluctantly she steps out from behind the tree and approaches.
+&lt;action&gt;Reluctantly she steps out from behind the tree and approaches.&lt;/action&gt;
 
+&lt;choices&gt;
 [[Push it off!|C1P63V1]]&lt;br&gt;
 [[Use the knife!|C1P63V2]]&lt;br&gt;
 [[Get me the knife!|C1P63V3]]
+&lt;/choices&gt;
```

Comment:
The `<br>`s are from before the templating engine, just remove them when you see them thanks. Also indenting 👍 
```suggestion
&lt;choices&gt;
  [[Push it off!|C1P63V1]]
  [[Use the knife!|C1P63V2]]
  [[Get me the knife!|C1P63V3]]
&lt;/choices&gt;
```